### PR TITLE
Update android backend.

### PR DESCRIPTION
**Pull request by dvc94ch**
This adds basic support for android to glutin. The android build failure is due a too old version of `cargo-apk`, the docker image needs to use https://github.com/rust-windowing/android-rs-glue/pull/256

### CODE REVIEW

File: glutin/src/api/egl/mod.rs [hunk 1]
```diff
@@ -621,6 +623,11 @@ impl Context {
                 egl.GetError()
             )
         }
+        if !self.vsync {
+            if egl.SwapInterval(self.display, 0) == ffi::egl::FALSE {
+                panic!("finish_impl: eglSwapInterval failed: 0x{:x}", egl.GetError());
```
Comment (goddessfreya) [inline glutin/src/api/egl/mod.rs]: Nit: `finish_impl` -> `on_surface_created`.

**Comment by katyo** [comment]
@dvc94ch I tested winit with glutin from android branch with my simple gl triangle test. This builds and works good. But I still have some troubles with glutin on some devices. Particularly, it fails on eglMakeCurrent with EGL_BAD_MATCH.

**Comment by dvc94ch** [comment]
Cool! I think it's maybe due to the orientation handling code. Is it a glutin example? Or do you have the code somewhere?

**Comment by katyo** [comment]
@dvc94ch This is my own [experimental](https://github.com/katyo/mixui/tree/master/demo) code.

**Comment by dvc94ch** [comment]
Closing in favor of #1274

**Pull request closed without merging**